### DISCUSSION TITLE
Filebeat Module Clarification

### DISCUSSION THREAD

**Discussion by syst3m10** [Security-Onion-Solutions/securityonion#4705] [Unsupported Versions] [answered]
Hello - I was hoping to get some clarification with the filebeat module configuration introduced in 2.3.60.

I am attempting to bring in Cisco ASA logs. I have modified the searchmanager.sls file under '/opt/so/saltstack/local/pillar/minions/'. I was mindful of correct spacing. I added the proper values according to the official filebeat cisco ASA module help (enabled: true, var.syslog_host, var.syslog_port, var.log_level). I then created a new firewall portgroup and hostgroup and added a minion firewall rule to allow traffic on UDP port 9003 (the port used when enabling the cisco module). 

After a salt state apply, I can see on the manager node port 9003 listening over UDP on interface 0.0.0.0 which is expected. I can see the firewall rules have successful applied when viewing iptables. I see no errors in the filebeat log files under /opt/so/log.  If i view the third_party_modules.yaml in the filebeat container i can see cisco is enabled. However, no logs are ingested. If I point the ASA to the standard syslog port, the raw logs do come in without issue. 

I believe I'm probably missing a simple step and was hoping I could get clarification on the process.

Thank you!

**Comment by syst3m10** [comment, marked as answer]
This can be closed, i was able to get the logs to ingest.

Thank you!

**Reply by defensivedepth** [reply]
@syst3m10 great to hear! 

Would you mind posting what you ran into and how you fixed it?

**Reply by syst3m10** [reply]
Truthfully, im not certain what resolved the issue. After digging around, reading, rebooting a few containers i noticed a new index populated and the logs were coming in. 

I think your documentation was accurate and straightforward. Either i misconfigured something or a container (filebeat or logstash) had to be restarted after the salt state was applied.

I will say the filebeat module addition is amazing and really makes the onboarding of appliances much easier. Great job to you and the team.

**Reply by defensivedepth** [reply]
Ok great to hear, thanks!
